app/views/content/train-to-be-a-teacher/qualifications-you-need-to-teach.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: "What qualifications do I need to be a teacher?"
```

Comment:
Worth changing this title so it is no longer a question? But check re SEO first

NB title and h1 can be different - this might be more appropriate as the title, so just the h1 that needs updating?